site/overview/getting-started.md

```diff
@@ -20,48 +20,172 @@ head:
-This guide will help you get started with Account Kit by setting up your environment, creating a Light Account (a type of smart account implementation), and sending a User Operation from it. By the end of this guide, you'll have a basic understanding of how to use the SDK and where to look for more advanced use cases.
+This guide will help you get started with Account Kit by setting up your environment, creating a smart account, and sending a `UserOperation` on its behalf. By the end of this guide, you'll have a basic understanding of how to use the SDK and where to look for more advanced use cases.
 
-## Install the Packages
+## 1. Install the Packages
 
-In your project directory, you'll need to install the required packages:
+In your project directory, you'll need to run the following to install the required packages:
 
 ::: code-group
 
 ```bash [npm]
+npm init -y
+npm install -save-dev typescript
 npm install @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
 ```
 
 ```bash [yarn]
+yarn init -y
+yarn add -D typescript
 yarn add @alchemy/aa-alchemy @alchemy/aa-accounts @alchemy/aa-core viem
 ```
 
-Note that we're installing [viem](https://viem.sh/) as well. Viem contains helpful abstractions and modules that may come in handy when using Account Kit. Additionally, many Account Kit modules leverage viem themselves.
+::: tip Note
+
+We're installing [viem](https://viem.sh/) as well. Viem contains helpful abstractions and modules that will come in handy when using Account Kit. Additionally, many Account Kit modules use `viem` themselves.
 
 :::
 
-## A Simple Light Account Example
+Make sure your new `package.json` file and `tsconfig.json` files look like the following. Note that we added `"type": module` in `package.json` for this example:
 
-Using the SDK, we'll deploy a Light Account and send a User Operation from it. The Light Account will be owned by an Externally Owned Account (EOA). Here's a demonstration:
+::: code-group
 
-<<< @/snippets/light-account.ts
+```json [package.json]
+{
+  "name": "account-kit-test",
+  "version": "1.0.0",
+  "description": "An Example Using Account Kit",
+  "main": "index.ts",
+  "type": "module",
+  "license": "ISC",
+  "devDependencies": {
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "@alchemy/aa-accounts": "^1.1.0",
+    "@alchemy/aa-alchemy": "^1.1.0",
+    "@alchemy/aa-core": "^1.1.0",
+    "viem": "^1.19.3"
+  }
+}
+```
 
-This initializes a `provider` for your smart account which is then used to send user operations from it. It also logs the address of the deployed smart account.
+```json [tsconfig.json]
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "module": "ES2022",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "skipLibCheck": true
+  }
+}
+```
 
-::: tip Note
-Remember to:
+:::
+
+You'll also want to make sure your `Node` version is _18.10.0_ using your version manager:
```

Comment:
isn't this being changed to 18.16?

Reply:
After aa-signers will change to 18.16, not needed right now